apps/prairielearn/src/tests/sprocs/users_select_or_insert.test.js

```diff
@@ -67,6 +77,7 @@ describe('sproc users_select_or_insert tests', () => {
       name: 'J.R. User',
     };
 
+    // Errors if passed in institution_id '1', does not match policy -- do we need another test?
```

Comment:
This is expected. An institution ID should only be passed to `users_select_or_insert` if we should limit the authentication to a specific institution, which in turn should only be true if the institution has a `uid_regexp` set. Institution 1 is the default institution and thus will not (and *cannot*) have a `uid_regexp` set.